app/assets/javascripts/snowcrash/modules/issues/table.js.coffee

```diff
@@ -119,6 +125,38 @@ class IssueTable
           $($row.find('td')[that.tagColumnIndex]).replaceWith("<td class='text-error'>Please try again</td>")
       }
 
+  # get selected issues ids and their titles, and build a form with that info
+  # inside the combine issues modal
+  onCombineSelected: (event) ->
+    $('#modal_combine_issues #issues_to_combine').empty()
+
+    issues_to_combine = []
+    $('.js-tbl-issues').find('input[type=checkbox]:checked.js-multicheck').each ->
+      $row = $(this).parent().parent()
+
+      id = @.name.split('_')[1]
+      name = $($row.find('td')[1]).find('a').html()
+      issues_to_combine.push({id: id, name: name})
+
+    for issue in issues_to_combine
+      html =  "<div class='radio'><label><input type='radio' name='id' value='#{issue.id}' />#{issue.name}</label></div>"
+      html += "<input type='hidden' name='sources[]' value='#{issue.id}' />"
+      $('#modal_combine_issues #issues_to_combine').append(html)
+
+    $("#modal_combine_issues").modal()
+
+    # TODO: $($row.find('td')[2]).replaceWith("<td class=\"loading\">Combining...</td>")
```

Comment:
removing, because we are not combining issues by ajax, we are using a full http request with page reloading